app/Http/Controllers/Airlines/DestroyAirlineController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Airlines;
+
+use App\Models\Airline;
+use Flugg\Responder\Contracts\Responder;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\JsonResponse;
+
+class DestroyAirlineController
+{
+    public function __invoke(int $id, Responder $responder): JsonResponse
+    {
+        try {
+            $airline = Airline::findOrFail($id);
+            $airline->delete();
+            return $responder->success()->respond();
+        } catch (ModelNotFoundException) {
+            return $responder->error(JsonResponse::HTTP_NOT_FOUND, 'Airline not found.')->respond();
+        }
+    }
```

Comment:
```suggestion
    public function __invoke(Airline $airline, Responder $responder): JsonResponse
    {
        $airline->delete();
        
        return $responder
            ->success()
            ->respond(JsonResponse::HTTP_NO_CONTENT);
    }
```
With Route Model Binding, you can do this. I personally use the `JsonResponse::HTTP_NO_CONTENT` status when the endpoint doesn't respond with anything.